client/js/app/stores/ExplorerStore.js

```diff
@@ -128,7 +128,14 @@ function _getDefaultFilterCoercionType(explorer, filter) {
 function _prepareUpdates(explorer, updates) {
   // TODO: We're assigning the response object directly onto the model so we
   // don't have to loop through the (sometimes) massive response object.
-  var newModel = _.merge({}, explorer, _.omit(updates, 'response'));
+  function customizer(objValue, srcValue, key, object, source) {
+    if (_.isArray(objValue)) {
+      return srcValue;
+    } else if (key === 'time' && _.isPlainObject(objValue)) {
+      return srcValue;
+    }
+  }
+  var newModel = _.mergeWith({}, explorer, _.omit(updates, 'response'), customizer);
```

Comment:
I'm getting an error that `mergeWith` is not a function with the latest version of lodash.